docs/content/en/docs/osmgmt/artifacts.md

```diff
@@ -33,6 +35,7 @@ EKS Anywhere does not distribute Ubuntu or RHEL OS images.
 However, see [Building node images]({{< relref "#building-node-images">}}) for information on how to build EKS Anywhere images from those Linux distributions.  Note:  if you utilize your Admin Host to build images, you will need to review  the DHCP integration provided by Libvirtd and ensure it is disabled.  If the Libvirtd DHCP is enabled, the "boots container" will detect a port conflict and terminate.
 
 ### Bottlerocket OS images for Bare Metal
+**_NOTE:_** [Starting with the v0.19 EKS Anywhere release, we are deprecating Bottlerocket support for bare metal](https://github.com/aws/eks-anywhere/issues/7754) )
```

Comment:
I think it's redundant to add the same link twice on the same page. This can be removed from here

Reply:
Agreed and thank you for your insight.